### PR TITLE
Support serverless.yml and serverless.yaml in getBindPath()

### PR DESCRIPTION
**Pull request by kichik**
This should fix the issue reported by @mrpgraae in #210.
I've also added support for `SLS_DEBUG` so we can more easily get some information on future errors.

### CODE REVIEW

File: lib/docker.js [hunk 1]
```diff
@@ -32,24 +34,43 @@ function buildImage(dockerFile) {
...
+ * Find a file that exists on all projects so we can test if Docker can see it too
+ * @param {string} servicePath
+ * @return {string} file name
+ */
+function findTestFile(servicePath) {
+  if (fse.pathExistsSync(path.join(servicePath, 'serverless.yml'))) {
+    return 'serverless.yml';
+  }
+  if (fse.pathExistsSync(path.join(servicePath, 'serverless.yaml'))) {
+    return 'serverless.yaml';
```
Comment (dschep) [inline lib/docker.js]: Did serverless drop support for `serverless.json`?

Comment (kichik) [reply lib/docker.js]: I was not aware that's an option. Let me add that.

File: lib/docker.js [hunk 2]
```diff
@@ -32,24 +34,46 @@ function buildImage(dockerFile) {
...
     'alpine',
     'ls',
-    '/test/serverless.yml'
+    `/test/${testFile}`
   ];
   try {
     const ps = dockerCommand(options);
-    return ps.stdout.trim() === '/test/serverless.yml';
+    if (process.env.SLS_DEBUG) {
+      console.log(`Trying bindPath ${bindPath} (${options})`)
```
[the remark this answers is not in the document]
Comment (kichik) [reply lib/docker.js]: I can get to it later this week. We should consider passing `this` as the second argument to `.map()` in `installAllRequirements()` to make this simpler and so we don't have to pass `this.serverless` around.

[the remark this answers is not in the document]
Comment (dschep) [reply lib/docker.js]: I like that idea! Get to it when you get to it :+1:

[the remark this answers is not in the document]
Comment (kichik) [reply lib/docker.js]: It ended up being a bigger surgery than I expected as we don't always call these functions with `.map()` and so `this` in that case is the global context. We might have to create a class to make `this` make sense there.

**Pull request merged**